app/code/Magento/Msrp/Test/Unit/Pricing/MsrpPriceCalculatorTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Msrp\Test\Unit\Pricing;
+
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Type as Type;
```

Comment:
I'm not sure this alias is relevant :)
```suggestion
use Magento\Catalog\Model\Product\Type as ProductType;
```